packages/dev/inspector/src/components/actionTabs/tabs/propertyGrids/animations/curveEditor/graph/playHeadComponent.tsx

```diff
@@ -55,25 +72,41 @@ export class PlayHeadComponent extends React.Component<IPlayHeadComponentProps,
         });
 
         this.props.context.onMoveToFrameRequired.add((frame) => {
+            if (!this._isMounted) {
+                return;
+            }
+
             this.props.context.moveToFrame(frame);
             this._moveHead(frame);
         });
 
         this.props.context.onGraphMoved.add((x) => {
+            if (!this._isMounted) {
+                return;
+            }
+
             this._offsetX = x;
             this.forceUpdate();
 
             this._moveHead(this.props.context.activeFrame);
         });
 
         this.props.context.onGraphScaled.add((scale) => {
+            if (!this._isMounted) {
+                return;
+            }
+
             this._viewScale = 1 / scale;
             this.forceUpdate();
 
             this._moveHead(this.props.context.activeFrame);
         });
     }

```

Comment:
I understand this is not entirely the meaning of this PR, but if you need to check if the component is mounted, it is registered in the wrong place. Instead of tracking whether or not the component is mounted, add those observers in the  componentDidMount function (and removed when it is unmounted).

Reply:
No problem, I think isMounted was there just to avoid calling the observer when it wasn't mounted, so I'm just adding/removing them now :)